libraries/standard/mqtt/include/mqtt.h

```diff
@@ -138,18 +138,40 @@ MQTTStatus_t MQTT_Connect( MQTTContext_t * const pContext,
                            const MQTTPublishInfo_t * const pWillInfo,
                            bool * const pSessionPresent );
 
+/**
+ * @brief Sends MQTT SUBSCRIBE for the given list of topics to the broker.
+ *
+ * @param[in] pContext Initialized MQTT context.
+ * @param[in] pSubscriptionList List of MQTT subscription info.
+ * @param[in] pSubscriptionList The number of elements in pSubscriptionList.
+ * @param[in] packetId packet ID generated by #MQTT_GetPacketId.
+ *
+ * @return See #MQTTStatus_t.
```

Comment:
The return values used seem to be `MQTTBadParameter`, `MQTTNoMemory`, `MQTTSendFailed`, and `MQTTSuccess`.

Reply:
Good catch. thanks